polaris-react/src/components/PositionedOverlay/utilities/math.ts

```diff
@@ -20,133 +28,210 @@ export function calculateVerticalPosition(
   fixed: boolean | undefined,
   topBarOffset = 0,
 ) {
```

Comment:
Make TS happy in PositionedOverlay.tsx

```suggestion
: {
  top?: number;
  bottom?: number;
  height: number;
  positioning: string;
} 
```